Tests/Editor/AvatarLoaderTests.cs

```diff
@@ -11,7 +13,8 @@ namespace ReadyPlayerMe.Core.Tests
     public class AvatarLoaderTests
     {
         private GameObject avatar;
-        
+        private GameObject avatar2;
```

Comment:
This is actually not necessary, you can just re-use the same avatar object field, overriding it once you have the vertex count of the first loaded avatar.